src/core/scene/a-scene.js

```diff
@@ -64,8 +63,8 @@ module.exports = registerElement('a-scene', {
         this.addEventListener('render-target-loaded', function () {
           this.setupRenderer();
           this.resize();
+          this.addFullScreenStyles();
```

Comment:
This is required so tests pass. Otherwise it will try to set canvas style without a canvas.